### ISSUE TITLE
FRC 2023 RoboRIO toolchain binaries require >=GLIBC_2.32 on Linux

### ISSUE THREAD

**Issue by peter9477** [rockwayrobotics/robot-on-rust#2]
I was trying `cargo build` in my WSL (Window Subsystem for Linux) with the stock 2023 toolchain binaries, and they complain that they want GLIBC_2.32, 2.33, or 2.34.  Unfortunately I'm using Debian 11 Bullseye there and it's apparently on 2.31.  

There's a ChiefDelphi thread noting they used Ubuntu 22.04 to build and it has 2.32.  Unfortunate they choose something so new, as there are lots of hosts with older stuff.

Anyway, I'm trying to build the toolchain from source in case that helps... currently it's been running for an hour and I feel like it might be only 2% done...  :(

**Comment by JoeWildfong** [comment]
Well that's unfortunate. You could try using the 2022 toolchain. We only need the linker so there's a good chance it'll work. Should be a one-line change in .cargo/config.toml to use a different linker (or there's probably a cargo argument as well).

**Comment by JoeWildfong** [comment]
This issue might fix this too: https://github.com/wpilibsuite/opensdk/issues/52

**Comment by peter9477** [comment]
Yeah, it didn't register that you were using the 2022 toolchain.  I already had 2023 installed from something earlier, and just set the path to point to that without noticing.  Anyway the 2022 build finally worked and both it and the stock one run fine on Bullseye.

Cargo build now gives me a >5MB arfur-robot binary.  Release mode shrinks that to under 400K....
